recipes/boost/all/conanfile.py

```diff
@@ -247,6 +263,28 @@ def config_options(self):
             # Shared builds of numa do not link on Visual Studio due to missing symbols
             self.options.numa = False
 
+        if tools.Version(self.version) >= "1.76.0":
+            # Starting from 1.76.0, Boost.Math requires a c++11 capable compiler
+            # ==> disable it by default for older compilers or c++ standards
+
+            def disable_math():
+                super_modules = self._all_super_modules("math")
+                for smod in super_modules:
+                    try:
+                        setattr(self.options, "without_{}".format(smod), True)
+                    except ConanException:
+                        pass
```

Comment:
just to be sure to understand:
- super modules is the complete list of modules which might have math in their own dependency graph?
- what could fail in `setattr(self.options, "without_{}".format(smod), True)`? Is not under control?

Reply:
> just to be sure to understand:
> 
>     * super modules is the complete list of modules which might have math in their own dependency graph?

Yes, the current implementation is quick and dirty and I don't guarantee it working under all combinations though.

>     * what could fail in `setattr(self.options, "without_{}".format(smod), True)`? Is it not under control?

The `dependencies` section of the `dependencies-x.yy.yml` file contains modules that are not valid options of boost.build (`configure_options` section). So I just ignore these.